app/src/main/java/com/example/pocketcloset/Fragments/WardrobeFragment.java

```diff
@@ -1,65 +1,224 @@
 package com.example.pocketcloset.Fragments;
 
 import android.os.Bundle;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.ToggleButton;
 
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
 
 import com.example.pocketcloset.R;
+import com.example.pocketcloset.adapters.ClothingAdapter;
+import com.example.pocketcloset.models.Clothing;
+import com.example.pocketcloset.models.User;
+import com.google.android.material.button.MaterialButtonToggleGroup;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
 
-/**
- * A simple {@link Fragment} subclass.
- * Use the {@link WardrobeFragment#newInstance} factory method to
- * create an instance of this fragment.
- */
 public class WardrobeFragment extends Fragment {
 
-    // TODO: Rename parameter arguments, choose names that match
-    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-    private static final String ARG_PARAM1 = "param1";
-    private static final String ARG_PARAM2 = "param2";
+    protected ClothingAdapter adapter;
+    protected List<Clothing> allClothing;
 
-    // TODO: Rename and change types of parameters
-    private String mParam1;
-    private String mParam2;
+    private SwipeRefreshLayout swipeContainer;
+    private RecyclerView rvWardrobe;
+    public ParseUser user =  User.getCurrentUser();
+    public MaterialButtonToggleGroup toggleButton;
 
-    public WardrobeFragment() {
+    public WardrobeFragment(ParseUser userToFilterBy) {
```

Comment:
This is a fragment. Fragments should have empty constructors. Arguments that are sent to Fragments should be done using `Bundle` and `setArguments`. Take a look [here](https://stackoverflow.com/questions/5425568/how-to-use-setarguments-and-getarguments-methods-in-fragments)